mne_bids/tests/test_utils.py

```diff
@@ -357,8 +355,9 @@ def test_find_matching_sidecar(return_bids_test_dir):
     # Now find a sidecar
     sidecar_fname = _find_matching_sidecar(bids_basename, bids_root,
                                            'coordsystem.json')
-    expected_file = op.join('sub-01', 'ses-01', 'meg',
-                            'sub-01_ses-01_coordsystem.json')
+    expected_file = op.join(
+        'sub-01', 'ses-01', 'meg',
+        'sub-01_ses-01_task-testing_run-01_coordsystem.json')
```

Comment:
Need this change if `_gen_bids_basename()` takes in bidspath instead of what it was originally in https://github.com/mne-tools/mne-bids/pull/446/files#r440344654

Reply:
humm, can't you do:

```py
bids_path.update(task=None, run=None)
_gen_bids_basename(bids_path)
```
?

Reply:
Updated accordingly.